templates/docs/patterns/icons.md

```diff
@@ -266,6 +315,13 @@ If you use a limited set of icons you may want to include them individually to r
 @include vf-p-icon-email;
 @include vf-p-icon-sizes;
 @include vf-p-icon-in-button;
+@include vf-p-icon-chevron-up;
```

Comment:
I would separate the application set with a line comment `// application icons` to have a clearer distinction between those.